code/game/machinery/floodlight.dm

```diff
@@ -145,7 +120,40 @@
 	update_icon(UPDATE_ICON_STATE)
 	return TRUE
 
+/obj/machinery/floodlight/crowbar_act(mob/living/user, obj/item/I)
+	if(!unlocked)
+		to_chat(user, "The cover is screwed tightly down")
+		return TRUE
+
+	if(!I.use_tool(src, user, volume = I.tool_volume))
+		return
+
+	if(open)
+		to_chat(user, "you pry the panel closed")
+		open = FALSE
+	else
+		to_chat(user, "you pry the panel open")
+		open = TRUE
+	update_icon(UPDATE_ICON_STATE)
+	return TRUE
+
+/obj/machinery/floodlight/wrench_act(mob/living/user, obj/item/I)
+	. = TRUE
+	if(!I.tool_use_check(user, 0))
+		return
+	default_unfasten_wrench(user, I)
+
 /obj/machinery/floodlight/extinguish_light(force = FALSE)
 	on = FALSE
 	set_light(0)
 	update_icon(UPDATE_ICON_STATE)
+
+/obj/machinery/floodlight/examine(mob/user)
+	. = ..()
+	if(!unlocked)
+		. +="<span class='notice'>The panel is <b>screwed</b> shut."
+	else
+		if(open)
+			. +="<span class='notice'>The panel is <b>pried</b> open, looks like you could fit a cell in there."
+		else
+			. +="<span class='notice'>The panel looks like it could be <b>pried</b> open, or <b>screwed</b> shut."
```

Comment:
```suggestion
	if(!unlocked)
		. +="<span class='notice'>The panel is <b>screwed</b> shut.</span>"
	else
		if(open)
			. +="<span class='notice'>The panel is <b>pried</b> open, looks like you could fit a cell in there.</span>"
		else
			. +="<span class='notice'>The panel looks like it could be <b>pried</b> open, or <b>screwed</b> shut.</span>"
```